temboardagent/sharedmemory.py

```diff
@@ -160,23 +167,24 @@ def check_uniqueness(self, worker, parameters):
         """
         for i in range(0, self.size):
             if len(self.commands[i].commandid) == T_COMMANDID_SIZE and \
-                self.commands[i].worker == worker and \
-                self.commands[i].parameters == parameters and \
-                self.commands[i].state != COMMAND_DONE and \
-                self.commands[i].state != COMMAND_ERROR:
+                    self.commands[i].worker == worker and \
+                    self.commands[i].parameters == parameters and \
+                    self.commands[i].state != COMMAND_DONE and \
+                    self.commands[i].state != COMMAND_ERROR:
```

Comment:
Can you try use 3 spaces here instead of 7 (ie. align with the "l" of "len") and see if Flake8 still complains?